### PR TITLE
Experimental flag

### LINKED ISSUE #30: Mark non-hidden entries with "stable"
@dgozman :
> Entries marked "hidden" are meant to change at any time. I'd mark non-hidden entries with "stable" sign or something.

### PR DESCRIPTION
**Pull request by paulirish**
image

Fixes #30 

plz review @dgozman @pavelfeldman

### CODE REVIEW

**Comment by paulirish** [comment]
Previewable at https://paulirish.github.io/debugger-protocol-viewer/

Plz take a look around to confirm things are tagged right.

**Comment by kdzwinel** [comment]
Thought you were against adding it ;)

1. Isn't that a little weird that some domains are marked as experimental while only some of their methods/types are marked as experimental? (e.g. [CSS.getPlatformFontsForNode](https://paulirish.github.io/debugger-protocol-viewer/tot/CSS/#method-getPlatformFontsForNode) ). What's the difference between non-experimental method in experimental domain vs experimental method in experimental domain?

2. Deprecated domains should link to their successor. As far as I remember, we have that info in protocol.json.

3. We should probably explain what these badges mean somewhere.

4. If domain is deprecated/experimental we can mark it as such in the menu.

File: styles/protocol.css [hunk 1]
```diff
@@ -1,6 +1,10 @@
+:root {
+    --font-stack: 'Roboto', 'Helvetica Neue', Helvetica, Arial, sans-serif;;
```
Comment (kdzwinel) [inline styles/protocol.css]: `;;`

File: styles/protocol.css [hunk 2]
```diff
@@ -182,18 +201,24 @@ a.reflink {
 }
 
 .permalink {
-  display: none;
+  /* display: none; */
```
Comment (kdzwinel) [inline styles/protocol.css]: lets remove it

File: _includes/method.html [hunk 3]
```diff
@@ -2,12 +2,11 @@
 {% capture permalink %}{{ site.baseurl }}{{ page.url }}#method-{{method.name}}{% endcapture %}
 
 <div class="method">
-  <h4 class="method-name monospace" id="method-{{method.name}}">{{method.name}}<a href="{{permalink}}" class="permalink">#</a></h4>
+  <h4 class="method-name monospace" id="method-{{method.name}}">{{method.name}}<a href="{{permalink}}" class="permalink">#</a> {% include experimental.html item=method %} </h4>
 
   <p class="method-description">
-    {{method.description}}
+    {{method.description}}
```
Comment (kdzwinel) [inline _includes/method.html]: trailing space

File: _includes/type.html [hunk 4]
```diff
@@ -4,9 +4,13 @@
 <div class="type">
   <h4 class="type-name monospace" id="type-{{type.id}}">{{type.id}}<a href="{{permalink}}" class="permalink">#</a></h4>
 
-  <p class="type-description">{{type.description}}</p>
+  <p class="type-description">
+    {{type.description}}
+    {% include experimental.html item=type %}
```
Comment (kdzwinel) [inline _includes/type.html]: it's a bit inconsistent that methods have the badge on the title level while types have it on the description level

screen shot 2016-08-18 at 23 59 54
screen shot 2016-08-19 at 00 01 02

File: styles/protocol.css (hunk 2 above)
Comment (paulirish) [inline styles/protocol.css]: the experimental badge is next to the #, so when it toggles display it moves the position of the badge.

File: _includes/type.html (hunk 4 above)
Comment (paulirish) [inline _includes/type.html]: Yah I agree. I'd rather all three were in description. 

I'm OK with Domain having a special treatment though.

**Comment by paulirish** [comment]
addressed feedback.

https://paulirish.github.io/debugger-protocol-viewer/ is updated.

File: _includes/type.html (hunk 4 above)
Comment (paulirish) [inline _includes/type.html]: fixed.

**Comment by paulirish** [comment]
> Isn't that a little weird that some domains are marked as experimental while only some of their methods/types are marked as experimental? 

yup. i now tag the domain in red and then ignore any subitems that are marked experimental

image

> Deprecated domains should link to their successor. 

there's only Console right now.. and the description indicates the successor.

File: _includes/type.html (hunk 4 above)
Comment (kdzwinel) [inline _includes/type.html]: Have you updated the preview? It still looks like experimental badge is on the title level for methods but on the description level for everything else:

screen shot 2016-09-01 at 23 01 37

https://paulirish.github.io/debugger-protocol-viewer/tot/DOM/

**Comment by kdzwinel** [comment]
LGTM! Love the red outline on the experimental domain box ✨

File: _includes/type.html (hunk 4 above)
Comment (paulirish) [inline _includes/type.html]: true. :)

okay now its fixed. thx for the catch.

**Pull request merged**